strings/quest_workers.json

```
@@ -1,116 +1,115 @@
 		"OOOOOOOOOOOOOOOOOOOOOoooooooooooooooooo!! ",
-		"The cargo has been received in supreme condition. Wish it was always like that, and not how it usually goes... ",
-		"Delivery received, but one of our fools got their finger torn off during unpacking.",
-		"Finally, a man at work.",
-		"I'll report your helpful work to the higher ups. ",
-		"AVE CARGONIA",
-		"Gotta work again…",
-		"I won't forget this, my dudes. This delivery might even get me a promotion. ",
-		"It actually came, it's actually intact. We were getting worried for a moment there.",
-		"Hurry up next time, you just barely made it in time. ",
-		"Work faster next time, our schedule is tight. ",
-		"Where's the damn beer?? Could you not put it in???? ",
-		"Well, I ain't getting fired today."
+		"Груз прибыл в целостности и сохранности, всегда бы так, а не как обычно... ",
+		"Доставку приняли, только у нас одному дурачку палец оторвало во время вскрытия контейнера.",
+		"Наконец-то, мужчина за работой.",
+		"Я отрапортую куда надо что вы нам очень помогли сегодня. ",
+		"АВЕ КАРГОНИЯ!!!!!!!!!!!!!",
+		"Опять вкалывать...",
+		"Мужики, век не забуду, эта доставка поможет мне получить должность получше. ",
+		"Реально пришло, реально всё на месте. А мы уже начали волноваться.",
+		"В следующий раз поторопитесь, едва успели. ",
+		"Работайте быстрее в следующий раз, у нас график срывается. ",
+		"А пиво где?? Вы чё его не положили??? ",
+		"Ну, сегодня меня не уволят."
 	],
 
 	"fast_complete_phrases": [
-		"DAMN YOU GOOD, MAN",
-		"Much respect for the speed, by the way.",
-		"You did really good, excellent job, guys. ",
-		"CARGONIA AVE",
-		"We'll provide a discount on some of our merchandise for your efficiency. ",
-		"You are to be rewarded for your urgency. ",
-		"In a blink of an eye, that's some skill. ",
-		"Hey, catch me later, I'll buy you a beer. ",
-		"You guys are MAD, can I work for you? ",
-		"I TOLD my guys you weren't going to let us down. ",
-		"I'll provide you with a discount for your efforts. ",
-		"You can expect some bonuses, you did good. ",
-		"You can when you want to. ",
-		"By the way, is it true one can hook up with a Skrell on your station? ",
-		"I always knew you were not going to disappoint. ",
-		"The samples came in right as the tests were about to begin, expect a bonus.",
+		"МУЖИК НУ ТЫ КРУТ.",
+		"Кстати, за скорость отдельный респект.",
+		"Реально хорошо справились, молодцы парни. ",
+		"КАРГОНИЯ АВЕ!!!!",
+		"За скорость дам скидку на часть продукции. ",
+		"За спешку вам полагается отдельная надбавка. ",
+		"Я глазом моргнуть не успел, вот это уровень. ",
+		"Кстати, я должен тебе пиво. ",
+		"Во вы лютые, а можно к вам на работу? ",
+		"А я говорил своим что вы не подведёте. ",
+		"Сделаю вам скидку за труды.",
+		"Можете рассчитывать на поблажки, хорошо потрудились. ",
+		"Умеете же, когда хотите. ",
+		"Кстати, а правда, что у вас там со скреллками можно познакомиться?? ",
```

```suggestion
		"Можете рассчитывать на поблажки, хорошо потрудились.",
		"Умеете же, когда хотите.",
		"Кстати, а правда, что у вас там со скреллками можно познакомиться?",
```
И вот пробелы бы лишние постирать...